### ISSUE TITLE
Error: Constant COMPATIBLE_VERSION already defined

### ISSUE THREAD

**Issue by febriyan1302** [phalcon/phalcon-devtools#1006]
Please how to solve this error ?

### Error phalcon devtools :

```php
Phalcon DevTools (3.1.2)

#!/usr/bin/env php
PHP Notice:  Constant COMPATIBLE_VERSION already defined in /root/phalcon-devtools/bootstrap/autoload.php on line 106

Phalcon DevTools (3.1.2)

#!/usr/bin/env php
PHP Notice:  Constant COMPATIBLE_VERSION already defined in /root/phalcon-devtools/bootstrap/autoload.php on line 106

Phalcon DevTools (3.1.2)

#!/usr/bin/env php
PHP Notice:  Constant COMPATIBLE_VERSION already defined in /root/phalcon-devtools/bootstrap/autoload.php on line 106
PHP Fatal error:  Maximum recursion depth exceeded in /root/phalcon-devtools/bootstrap/autoload.php on line 117
```

### Details

* Phalcon Framework version: 3.1.2
* PHP Version: PHP 5.6.30
* Operating System: CentOS
* Server: Nginx

**Comment by sergeyklay** [comment]
@febriyan1302 Could you please provide steps to reproduce?

**Comment by selden1992** [comment]
me too

**Comment by febriyan1302** [comment]
Hi @sergeyklay , i don't know what happen, but today when i try to call #phalcon it works perfectly, btw thanks for your response.

![screen shot 2017-04-10 at 2 37 24 pm](https://cloud.githubusercontent.com/assets/18423871/24850756/77f834c2-1dfb-11e7-9e6b-777870b77a87.png)

**Comment by qjacquet** [comment]
Hi !
Have the same issue here, but `#phalcon` not works.

### Details
- Phalcon Framework version: 3.1.2
- PHP Version: PHP 7.1
- Operating System: Windows 7
- Server: Apache

**Comment by sergeyklay** [comment]
I'll try to sort out ASAP.

**Comment by sergeyklay** [comment]
Fixed in the 3.2.x branch.

Cc: @Jurigag

**Comment by shijij** [comment]
Just happened today.

To fix this, checkout the 3.2.x branch.

`git checkout 3.2.x`

**Comment by alejosv** [comment]
In my case, this error happen if a run the command `phalcon` into of phalcon-devtools folder. If I exit the phalcon-devtools folder and run again the command `phalcon` work fine.
